lib/utils/entity-utils.ts

```diff
@@ -26,7 +26,7 @@ export type EntityMap = Syntax.EntityMap;
 
 const MAX_SMALL_INTEGER = 12;
 
-const ENTITIES : { [key : string] : (idx : number) => AnyEntity } = {
+export const ENTITIES : { [key : string] : (idx : number) => AnyEntity } = {
```

Comment:
Why do you need to export this? Use `makeDummyEntities`, don't use this directly.